docker/Dockerfile.quinoa-build-travis

```diff
@@ -18,10 +16,13 @@ ARG TRAVIS_BRANCH
 ARG TRAVIS_JOB_NUMBER
 ARG TRAVIS_PULL_REQUEST
 ARG TRAVIS_JOB_ID
-ARG TRAVIS_TAG
 ARG TRAVIS_REPO_SLUG
 ARG TRAVIS_COMMIT
 
+ENV RUNNER_ARGS "--bind-to none -oversubscribe"
+ENV OMPI_MCA_plm isolated
+ENV PATH /opt/openmpi/${COMPILER}/bin:$PATH
```

Comment:
Nice!

Reply:
Yeah, since all builds now run inside the same image.